src/ts_catalog/continuous_agg.c

```diff
@@ -401,6 +395,89 @@ continuous_agg_formdata_fill(FormData_continuous_agg *fd, const TupleInfo *ti)
 		heap_freetuple(tuple);
 }
 
+/*
+ * Fill the fields of a integer based bucketing function
+ */
+static void
+continuous_agg_fill_bucket_function_ib(ContinuousAggsBucketFunction *bf, bool *isnull,
```

Comment:
```suggestion
cagg_fill_bucket_function_integer_based(ContinuousAggsBucketFunction *bf, bool *isnull,
```
Nit.

Reply:
Naming of these two functions is changed.